parser_example_test.go

```diff
@@ -16,24 +16,20 @@ package parser_test
 import (
 	"fmt"
 	"github.com/pingcap/parser"
-	_ "github.com/pingcap/tidb/types/parser_driver"
 )
 
 // This example show how to parse a text sql into ast.
 func Example_parseSQL() {
 
-	// 0. make sure import parser_driver implemented by TiDB(user also can implement own driver by self).
-	// and add `import _ "github.com/pingcap/tidb/types/parser_driver"` in the head of file.
```

Comment:
I don't think we want to recommend `test_driver` for general use. Perhaps move this entire file to `README.md` and keep the content unchanged.